force-app/main/default/aura/modalDialog/modalDialog.cmp

```diff
@@ -0,0 +1,41 @@
+<aura:component>
```

Comment:
this looks like a fine implementation but I think we should just use the existing overlay component!  =) https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/ref_lightning_overlayLibrary.htm